test/OpenTelemetry.Exporter.Prometheus.HttpListener.Tests/PrometheusHttpListenerTests.cs

```diff
@@ -35,38 +35,44 @@ public class PrometheusHttpListenerTests
         [InlineData("https://example.com")]
         [InlineData("http://127.0.0.1")]
         [InlineData("http://example.com", "https://example.com", "http://127.0.0.1")]
-        public void ServerEndpointSanityCheckPositiveTest(params string[] uris)
+        public void UriPrefixesPositiveTest(params string[] uriPrefixes)
```

Comment:
Could we add a couple cases here using `+` & `*` in the prefix?

Reply:
Let me try that now, I didn't coz I guess that might require higher privilege/access 😆

Reply:
Added in https://github.com/open-telemetry/opentelemetry-dotnet/pull/3521/commits/56d94c22c3465a6b4a3dfbd490d45eaa0dcb1abf